config/default/configmap/inferenceservice.yaml

```diff
@@ -62,11 +62,13 @@ data:
     }
   explainers: |-
     {
-        "image" : "docker.io/seldonio/alibiexplainer",
-        "defaultImageVersion": "0.2.3",
-        "allowedImageVersions": [
-           "0.2.3"
-        ]
+        "alibi": {
+            "image" : "docker.io/seldonio/alibiexplainer",
```

Comment:
Isn't this a gcr KFServing image?

Reply:
good point, probably Clive put this before image was published.